package.json

```diff
@@ -2,6 +2,6 @@
     "name": "com.arm.mobile-studio",
     "description": "Unity C# bindings for Mobile Studio.",
     "version": "1.0.0",
-    "unity": "2018.4",
+    "unity": "2020.3",
```

Comment:
Do we want to increase the minimum supported version this much?

Reply:
2020.3 is now the oldest currently supported LTS release, and the minimum version for the profiling API. We could make the statistics integration conditional at compile time if we want to keep compatibility for the older Unity versions.

Reply:
No, I think changing this makes sense then. Thanks!